src/status_im/contexts/profile/settings/events.cljs

```diff
@@ -91,6 +91,18 @@
              (boolean private?)
              {})))
 
+(rf/defn toggle-peer-syncing
```

Comment:
Since this is green/new code, you could follow the guideline https://github.com/status-im/status-mobile/blob/9fab422e0c9fa7c72bdc2dff8973785b768721ca/doc/new-guidelines.md#registering-event-handlers